src/main/java/bio/terra/cli/command/Main.java

```diff
@@ -116,20 +119,28 @@ public void run() {}
             .errors(CommandLine.Help.Ansi.Style.fg_red, CommandLine.Help.Ansi.Style.bold)
             .build();
 
```

Comment:
I'm not convinced of the value of aligning with the posix exit codes, because I don't think there are great analogs. Our 4 possible exit codes are: success, user-actionable exception, system exception, unexpected exception. None of the exceptions consistently map to the cause (e.g. docker connection error is a user-actionable exception, but so is an invalid argument). 

I'm certainly open to discussion here though if you or anyone else thinks I should reconsider. Let me know if you think this warrants a post stand-up discussion. If we choose to go the posix error code route, I do think we should change the mapping.`EAGAIN` means to try again, which I would not expect to fix a CLI `SystemException`.

Reply:
I'm fine with not using posix error codes, the thinking was just that scripting languages might be able to turn them into meaningful exceptions (I've used libraries to map the return codes of syscalls to exception types in python, for instance).

The bigger concern is just that the meaning is documented so that people using the tool programmatically can interpret the return codes, since there are more than one value.

Reply:
RE scripting languages auto-generating exceptions: Makes sense. I'm not sure what action the caller could take on catching one of these exceptions from the CLI though -- that's my hesitancy to mapping to posix.

RE documentation: Do I understand correctly then that you would vote for including exit codes in the main usage help? If so, I will include that change as an add-on to my current PR. Sorry, I think I just read past that in your original comment, and am now re-reading it.

Reply:
No worries...  including in either one or both of README.md and/or usage help would be fine with me.

Reply:
Sounds good. Created follow-on ticket [PF-704](https://broadworkbench.atlassian.net/browse/PF-704). Thanks!